.run/Runelite.run.xml

```diff
@@ -10,4 +10,4 @@
       <option name="Make" enabled="true" />
     </method>
   </configuration>
-</component>
+</component>
```

Comment:
there is still a change here, This file cannot be in the PR, please put it back to the original